modules/restful_example/src/Plugin/resource/Articles__1_0.php

```diff
@@ -0,0 +1,55 @@
+<?php
+
+/**
+ * @file
+ * Contains \Drupal\restful_example\Plugin\resource\Articles__1_0.
+ */
+
+namespace Drupal\restful_example\Plugin\resource;
+
+use Drupal\restful\Plugin\resource\ResourceEntity;
+use Drupal\restful\Plugin\resource\ResourceInterface;
+
+/**
+ * Class Articles
+ * @package Drupal\restful\Plugin\formatter
+ *
+ * @Resource(
+ *   name = "articles:1.0",
+ *   resource = "articles",
+ *   label = "Articles",
+ *   description = "Export the article content type with cookie authentication.",
+ *   authenticationTypes = TRUE,
+ *   authenticationOptional = TRUE,
+ *   dataProvider = {
+ *     "entityType": "node",
+ *     "bundles": {
```

Comment:
Yeah, that's the trickiest part. I have two ideas:
  1. If there is more than one different bundle, then `create` is disabled.
  2. We define some extra predefined key that allows you to select the appropriate bundle.

I like both options, but I see the difficulties. Let's see what you think of this.

Maybe it's worth mentioning that when posting to a resource that has embedded resources (ex: a node with tags) we are already allowing a special `'request'` key to provide info for the sub-request when posting the tags. That may be some precedence in favor of 2. Something like sending a payload like:
```php
// PUT request.
array(
  'title' => 'Article',
  'body' => 'Lorem ipsum',
  'tags' => array(
    'request' => array('method' => 'POST', …),
    'values' => array('alpha', 'beta'),
  ),
);
```